stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java

```diff
@@ -291,6 +293,15 @@ public synchronized void addAdditionalResult() throws IOException {
     public synchronized MasterBenchTaskResult getResult() {
       return mResult;
     }
+
+    private String getMethodDescription(String type, String methodName) {
+      if (type.contains("RPC")) {
+        final int classNameDivider = methodName.lastIndexOf(".");
```

Comment:
Could this return -1, and will that cause a problem?

Reply:
Yes it could return -1. No it wouldn't cause a problem because s.substring(0) returns s.

Reply:
Conversely, returning the last index is also not a problem because 

s.substring(s.length) returns an empty string.